### PR TITLE
Add category selection if a human player is active

### PR DESCRIPTION
**Pull request by Arden488**
Based on user story: https://trello.com/c/CsqdWW6l

This PR:
- adds category selection logic (from a human player point of view)
- adds some styling
- updates some previous related pieces

### CODE REVIEW

**Review by hughwin — APPROVED** [review]

**Review by nathanieldschneider — APPROVED** [review]
Please can you commit more frequently - your commits should tell a story of the changes you have made throughout a PR so we get a clear idea of what you have done. Otherwise, it looks good. ;)

File: src/main/resources/com/toptrumps/online/scripts/card.js [hunk 1]
```diff
@@ -24,8 +27,44 @@ const Card = (function() {
...
+        $attributesWrapper.addClass(cardAttributesWrapperActiveClass);
+
+        const $attributes = getAttributes(playerSelector);
+        $($attributes).on("click", function() {
+            const $target = $(this);
+            $($attributes).removeClass(cardAttributeActiveClass);
+            $target.addClass(cardAttributeActiveClass);
+
+            $(document).trigger("game:categorySelect", {
+                category: $target.data("category")
```
Comment (BelfDev) [inline src/main/resources/com/toptrumps/online/scripts/card.js]: Can we keep the "Attribute" semantics?

File: src/main/resources/com/toptrumps/online/scripts/game.js [hunk 2]
```diff
@@ -1,19 +1,77 @@
 const Game = (function() {
+    const NUMBER_OF_AI_PLAYERS = 4;
```
Comment (BelfDev) [inline src/main/resources/com/toptrumps/online/scripts/game.js]: This is temporary, right?

**Review by BelfDev — CHANGES_REQUESTED** [review]
The PR looks good! I'd just ask to keep with the "Attribute" semantics instead of "Category".

File: src/main/resources/com/toptrumps/online/scripts/card.js (hunk 1 above)
Comment (Arden488) [reply src/main/resources/com/toptrumps/online/scripts/card.js]: Updated throughout the code

File: src/main/resources/com/toptrumps/online/scripts/game.js (hunk 2 above)
Comment (Arden488) [reply src/main/resources/com/toptrumps/online/scripts/game.js]: Yes
I added TODO comment

**Review by BelfDev — APPROVED** [review]

**Pull request merged**